src/main/java/cl/transbank/webpay/WebpayOptions.java

```diff
@@ -1,16 +1,16 @@
 package cl.transbank.webpay;
 
+import cl.transbank.common.IntegrationType;
```

Comment:
Same comments of class `PatpassOptions` apply here

Reply:
done